mz_bokeh_package/environment.py

```diff
@@ -60,3 +60,28 @@ def get_error_page_url() -> str:
         mz_app_url = "error"
 
     return mz_app_url
+
+
+def get_db_host() -> str:
+    """Returns a database host address based on the current environment
+
+    The database is a Postgres database running on GCP.
+
+    Returns:
+        str: Database host address
+    """
+    STAGING_HOST = "35.233.124.184"
+    PRODUCTION_HOST = "104.199.26.69"
+
+    env = get_environment()
+
+    if env == 'staging':
+        host = STAGING_HOST
+    elif env == 'production':
+        host = PRODUCTION_HOST
+    elif env == 'dev':
+        host = os.getenv('DB_HOST', STAGING_HOST)
```

Comment:
Roi, we agreed that the default for dev will be localhost, and we'll allow users to override that with the DB_HOST environment.

Reply:
@oriyudilevich right!